CODING-CONVENTIONS.md

```diff
@@ -47,7 +47,7 @@ Orion embraces typical Java idioms including using an Object Oriented approach t
    - Don't pass lambdas into executors because it makes it harder to identify the threading interactions. The lambda makes the code shorter but not clearer. Instead use a separate class or extract a method. 
 * For good examples, refer to the APIs the JDK itself exposes. 
 
->**Note** If you're not sure what idiomatic Java looks like, start by following the typical patterns and naming used in Orion.
+>**Note** If you're not sure what idiomatic Java looks like, start by following the typical patterns and naming used in Orion, or otherwise, other Pegasys codebases.
```

Comment:
I'm not sure if Orion follows the same patterns defined by Pantheon or other projects. I'd rather say that we should stick to the patterns in Orion.

(and while we do that, we should probably decide what the patterns are) ;)